charts/kminion/values.yaml

```diff
@@ -50,6 +50,7 @@ service:
   type: ClusterIP
   port: 8080 # This port is also used as exposed container port
   annotations: {} # # Annotations to add to the service
+  extraPorts: []
```

Comment:
Could you provide a commented sample value here?

Reply:
done ... thanks for your time